### ISSUE TITLE
Can't seem to open .cbr files on Linux

### ISSUE THREAD

**Issue by logix2** [ollm/OpenComic#1]
Hi. I'm using OpenComic v0.1.1 on Ubuntu 19.04 and the application can't seem to be able to open .cbr files. I tried it with two .cbr files and it failed to open both of them, with one being added but unable to open (and it has no preview), and the other not even showing up in OpenComic.

I got this error in DevTools:

```
fs.js:121 Uncaught Error: ENOENT: no such file or directory, scandir '/tmp/opencomic/4b0c0da6043684fbd9025dc6cc7772e43742fd5d'
    at Object.readdirSync (fs.js:811)
    at Object.fs.readdirSync (electron/js2c/asar.js:660)
    at loadFilesIndexPage (/opt/OpenComic/resources/app.asar/scripts/dom.js:213)
    at /opt/OpenComic/resources/app.asar/scripts/dom.js:465
    at /opt/OpenComic/resources/app.asar/scripts/file-compressed.js:181
    at /opt/OpenComic/resources/app.asar/node_modules/node-unrar/lib/index.js:15
    at /opt/OpenComic/resources/app.asar/node_modules/node-unrar/lib/index.js:27
    at ChildProcess.exithandler (child_process.js:319)
    at ChildProcess.emit (events.js:194)
    at maybeClose (internal/child_process.js:998)
handleErrorFromBinding @ fs.js:121
readdirSync @ fs.js:811
fs.readdirSync @ electron/js2c/asar.js:660
loadFilesIndexPage @ /opt/OpenComic/resources/app.asar/scripts/dom.js:213
(anonymous) @ /opt/OpenComic/resources/app.asar/scripts/dom.js:465
(anonymous) @ /opt/OpenComic/resources/app.asar/scripts/file-compressed.js:181
(anonymous) @ /opt/OpenComic/resources/app.asar/node_modules/node-unrar/lib/index.js:15
(anonymous) @ /opt/OpenComic/resources/app.asar/node_modules/node-unrar/lib/index.js:27
exithandler @ child_process.js:319
emit @ events.js:194
maybeClose @ internal/child_process.js:998
(anonymous) @ internal/child_process.js:403
emit @ events.js:194
(anonymous) @ net.js:593
```

**Comment by ollm** [comment]
I have posted an realese with bug fixes with compressed files. In this version the error should not be present.

https://github.com/ollm/OpenComic/releases/tag/v0.1.2

**Comment by logix2** [comment]
While now CBR files open, double clicking on a page from a CBR doesn't open the page (like it does for CBZ files)... it doesn't do anything.

**Comment by ollm** [comment]
Do you see an error in DevTools when trying to open the page?

**Comment by logix2** [comment]
Yes, sorry I forgot to add it:

```
fs.js:121 Uncaught Error: ENOTDIR: not a directory, stat '/home/logix/Downloads/Batman - Universe 001 (2019) (Digital) (Zone-Empire).cbr/ComicInfo.xml'
    at statSync (fs.js:876)
    at Object.fs.statSync (electron/js2c/asar.js:293)
    at Object.openComic (/opt/OpenComic/resources/app.asar/scripts/dom.js:1443)
    at HTMLDivElement.onclick (index.html:1)
handleErrorFromBinding @ fs.js:121
statSync @ fs.js:876
fs.statSync @ electron/js2c/asar.js:293
openComic @ /opt/OpenComic/resources/app.asar/scripts/dom.js:1443
onclick @ index.html:1
fs.js:121 Uncaught Error: ENOTDIR: not a directory, stat '/home/logix/Downloads/Batman - Universe 001 (2019) (Digital) (Zone-Empire).cbr/ComicInfo.xml'
    at statSync (fs.js:876)
    at Object.fs.statSync (electron/js2c/asar.js:293)
    at Object.openComic (/opt/OpenComic/resources/app.asar/scripts/dom.js:1443)
    at HTMLDivElement.onclick (index.html:1)
handleErrorFromBinding @ fs.js:121
statSync @ fs.js:876
fs.statSync @ electron/js2c/asar.js:293
openComic @ /opt/OpenComic/resources/app.asar/scripts/dom.js:1443
onclick @ index.html:1
```

**Comment by ollm** [comment]
I fixed the error in 25c3cc53cbab8ffad7c4cc432a5ed204afb5815d

The correct path to the file was not used when checking if it was a directory or not, and this cause the error.

Thanks for the report.

A greeting.

**Comment by ollm** [comment]
I posted a release with the bug fixed.

https://github.com/ollm/OpenComic/releases/tag/v0.1.3
